src/server.js

```diff
@@ -374,6 +370,20 @@ function start(opts) {
     return res.redirect(301, '/styles/' + req.params.id + '/');
   });
   */
+  serveTemplate('/styles/:id/wmts.xml', 'wmts', function(req) {
+    var id = req.params.id;
+    var wmts = clone((config.styles || {})[id]);
+    if (!wmts) {
+      return null;
+    }
+    if(wmts.hasOwnProperty("serve_rendered")&!wmts.serve_rendered){
```

Comment:
Please fix code formatting here (space after `if`, `&` -> ` && ` (with spaces around), `){` -> `) {`, indentation of the return below, ...)